docs/content/concepts/annotation-context.md

```diff
@@ -72,7 +72,7 @@ By default, Rerun will automatically assign colors to each class id, but by defi
 you can explicitly determine the color of each class.
 
 * Python: [`rr.SegmentationImage`](https://ref.rerun.io/docs/python/HEAD/common/images/#rerun.SegmentationImag)
-* Rust: Log a [`Tensor`](https://docs.rs/rerun/latest/rerun/components/struct.Tensor.html) with [`TensorDataMeaning::ClassId`](https://docs.rs/rerun/latest/rerun/components/enum.TensorDataMeaning.html#variant.ClassId)
+* Rust: Log a [`rerun::SegmentationImage`](https://docs.rs/rerun/0.9.0-alpha.6/rerun/archetypes/struct.SegmentationImage.html)
```

Comment:
We often use 🐍 and 🦀 to prefix our Python and Rust links… though I don't know what we will use for C++ 😬

Reply:
🌊 i believe

Reply:
did we definitely decide whether it's not 🦭

Reply:
that question is not for 0.9 to settle :)